models/pod.go

```diff
@@ -112,6 +117,10 @@ func (pod *Pod) Parse(p *core_v1.Pod) {
 	_, pod.VersionLabel = p.Labels[conf.IstioLabels.VersionLabelName]
 }
 
+func isIstioProxy(pod *core_v1.Pod, container *core_v1.Container, conf *config.Config) bool {
+	return pod.Namespace == conf.IstioNamespace && (container.Name == "istio-proxy" || strings.HasPrefix(pod.Name, "istio-ingressgateway") || strings.HasPrefix(pod.Name, "istio-egressgateway"))
```

Comment:
My radar goes off when I see hard-coded names like this. Are we guaranteed that these are the names Istio always uses? Are these customizable by the user via istioctl? If so, we need to either (a) document that they MUST be named this for Kiali to work or (b) make these customizable in the Kiali CR also

Reply:
In short, I have no idea.  But the fallout is fairly minimal if these names don't match, and as such I'm not going to overblow this PR twith configurability.  If a real user has a real problem we can address it at that time.